kubeflow/Makefile

```diff
@@ -103,15 +83,10 @@ common/kubeflow-issuer \
 .PHONY: apply-v3
 apply-v3: validate-values check-name 
 	@for component in $(components) ; do \
-	$(MAKE) apply-component component_path=$$component || break ; \
+	BUILD_DIR=$(BUILD_DIR) NAME=$(NAME) KFCTXT=$(KFCTXT) LOCATION=$(LOCATION) PROJECT=$(PROJECT) ./hack/apply_component.sh -path $$component; \
 	done
 
-	# Kick the IAP pod because we will reset the policy and need to patch it.
-	# TODO(https://github.com/kubeflow/gcp-blueprints/issues/14)
-	kubectl --context=$(KFCTXT) -n istio-system delete pods -l service=iap-enabler
-	# Kick the backend updater pod, because information might be outdated after the apply.
-	# https://github.com/kubeflow/gcp-blueprints/issues/160
-	kubectl --context=$(KFCTXT) -n istio-system delete pods -l service=backend-updater
+	KFCTXT=$(KFCTXT) $(MAKE) -C common/iap-ingress pod-reset
```

Comment:
Thank you, probably good enough for now!
It will be helpful to identify why it needs to run at the end and which external dependencies it has and try to avoid this.

Reply:
Looks like we need to enforce the backend updater to reconcile for a healthy backend. Maybe we want to wait for iap-ingressgateway to be ready first.